Makefile

```diff
@@ -0,0 +1,29 @@
+#install kakarot
+
+install:
+	sh kakarotup
+
+# install dependencies, automatically creates a virtual environment
+poetry-install:
+	poetry install
+
+# run devnet
+devnet: poetry-install 
+	 poetry run starknet-devnet --seed 0 --disable-rpc-request-validation --load-path deployments/devnet/devnet.pkl --timeout 300
+
+# build
+build:
+	cargo build --all --release
+
+# run
+run: 
```

Comment:
it looks like given the current format for the `example.env` file, sourcing won't work. I would update the `example.env` and remove all spaces.
```
# Copy this file to .env and fill in the values
STARKNET_RPC_URL=http://0.0.0.0:5050/rpc
RUST_LOG=debug
KAKAROT_HTTP_RPC_ADDRESS=0.0.0.0:3030
KAKAROT_ADDRESS=0x1234
PROXY_ACCOUNT_CLASS_HASH=0x1234
```

Reply:
Oh! I already had an issue opened for it! Looks like I forgot to push the commit to fix it!

-> https://github.com/sayajin-labs/kakarot-rpc/issues/133